python/sglang/srt/sampling/penaltylib/penalizers/repetition_penalty.py

```diff
@@ -1,6 +1,7 @@
 from typing import List
 
 import torch
+from sgl_kernel import sampling_scaling_penalties
```

Comment:
import and use this when `torch.cuda.is_available() and torch.version.cuda` are true

please write that as a utility function

Reply:
done